src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/HeaderReader.cs

```diff
@@ -19,17 +21,10 @@ public HeaderReader(IDictionary<string, DataClassification> headersToLog, IRedac
     {
         _redactorProvider = redactorProvider;
 
-        _headers = headersToLog.Count == 0
-            ? Array.Empty<KeyValuePair<string, DataClassification>>()
-            : headersToLog.ToArray();
+        _headers = headersToLog.Count == 0 ? [] : headersToLog.ToArray();
```

Comment:
Is `[]` a new syntax for `Array.Empty<T>`?

Reply:
Yes